sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -303,7 +303,11 @@ skewSpec
     ;
 
 locationSpec
-    : LOCATION STRING
+    : LOCATION location=STRING
```

Comment:
nit: for such a simple rule, we may not need to create names. `LOCATION STRING` is good enough.